boards/arm/kite_nrf9160/board.c

```diff
@@ -0,0 +1,20 @@
+#include <zephyr/init.h>
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/gpio.h>
+
+#if !defined(CONFIG_REGULATOR)
+
+static int enable_regulator_at_boot(const struct device *unused)
```

Comment:
A comment here explaining that we're not REALLY using a regulator, just the handy bindings, and we don't recommend treating it as a regulator anyway?